sdk/core/core-paging/package.json

```
@@ -35,15 +35,22 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc -p .",
     "build:test": "tsc -p .",
-    "clean": "",
+    "check-format": "echo skipped",
+    "clean": "echo skipped",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
```

We should use the top level prettierrc:
```suggestion
     "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
```